style.css

```diff
@@ -0,0 +1,112 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap');
+
+* {
+  background: #1b115e;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
```

Comment:
- [ ] [ OPTIONAL ]To make changes to the suggestion of the previous review, Kindly use the ROBOTO style for your project to point the Figma design. 
      `font-family: 'Roboto', sans-serif;`